debug/mutex_test.go

```diff
@@ -65,21 +65,20 @@ func TestRWMutexReportEvery(t *testing.T) {
 	}()
 
 	// Wait for writer to appear
-	for {
-		if atomic.LoadInt64(&mutex.writers) != 1 {
-			time.Sleep(time.Millisecond)
-		}
-		break
+	for atomic.LoadInt64(&mutex.writers) != 1 {
```

Comment:
Will update this, the previous should have never had a `for` loop, at least it was useless.